includes/class-coblocks-block-assets.php

```diff
@@ -412,16 +412,9 @@ public function frontend_scripts() {
 		// Post Carousel block.
 		if ( $this->is_page_gutenberg() || has_block( 'coblocks/post-carousel' ) || has_block( 'core/block' ) ) {
 			wp_enqueue_script(
-				'coblocks-slick',
-				$vendors_dir . '/slick.js',
-				array( 'jquery' ),
-				COBLOCKS_VERSION,
-				true
-			);
-			wp_enqueue_script(
-				'coblocks-slick-initializer-front',
-				$dir . 'coblocks-slick-initializer-front.js',
-				array( 'jquery' ),
+				'coblocks-post-carousel',
+				$dir . 'coblocks-post-carousel.js',
+				array(),
```

Comment:
Notice where on like 368 of this file I still keep around old jQuery scripts for the masonry block. That is because people who are using old markup of this block but have not yet gone into the editor and deprecated the block will have a broken experience by updating CoBlocks. We must be careful to account for cases of poeple who have not yet deprecated. Perhaps eventually we will be able to competely drop these scripts and hope that most people are updated and deprecated.

Reply:
if the block deprecates correctly why would there need to be any manual effort on the user's part?

Reply:
or this would be for the front end

Reply:
@AnthonyLedesma i just tested this using a post carousel block on master. previewed the front end, and then switched branches to this branch and without loading the editor refreshed the front end and the carousel is still working. i believe because the markup that the post carousel is using a server side rendered markup the markup is automatically updated and the new front end script is controlling the new version of this block.